### ISSUE TITLE
Bane Nightmare getting stuck on Nightmare End

### ISSUE THREAD

**Issue by TheUnholyWrath** [ValveSoftware/Dota2-Gameplay#8883]
### Ability name

Nightmare

### Description

From https://www.reddit.com/r/DotA2/comments/1330jly/nightmare_did_not_return_back_to_normal/

https://user-images.githubusercontent.com/46859977/235408719-313df684-2833-4c2e-b017-99603483f38f.mp4

### Example Match ID (and possibly Timestamp)

7131097287

**Comment by commentator24** [comment]
you can simply press the skill to cancel it you know?

**Comment by anotherCoward** [comment]
@commentator24 look at the reddit post. Bane tried, it was not working.

**Comment by commentator24** [comment]
umm.. i tested both nightmare on both allies and enemies i can cancel it myself

**Comment by commentator24** [comment]
really not sure whats the issue here. you should close this
https://user-images.githubusercontent.com/109324155/235444326-07fa7bf3-2357-469d-b353-58939449587b.mp4

**Comment by TheUnholyWrath** [comment]
> really not sure whats the issue here. you should close this https://user-images.githubusercontent.com/109324155/235444326-07fa7bf3-2357-469d-b353-58939449587b.mp4

you have to attack the bane to wake him up out of the sleep, while the bkb and lotus are still both active. you're not actually replicating it

**Comment by anotherCoward** [comment]
Very simple to reproduce

1. Bane uses `Nightmare` on any friendly Unit
2. Enemy uses `BKB` on himself
3. Enemy uses `Lotus Orb` on himself
4. Enemy attacks the nightmared unit
5. Enemy attacks Bane before he has the chance to use `Nightmare End`
6. Bane can't use `Nightmare` anymore and sticks around with `Nightmare End` for the rest of the match.

Even if he tries to cast `Nightmare End` while the effect is still active on the BKB'd Enemy, it won't change the Skill back to `Nightmare`.

**Comment by travis4dams** [comment]
I had the same issue in a game last night (I was Bane).  Persisted for the rest of the game and even after I died a couple more times.

Match ID: 7133444534
Timestamp: 35:59

Video clip attached.

https://user-images.githubusercontent.com/39527625/235468436-1087a89a-b1f2-4404-b991-b31a822f61b6.mp4

**Comment by commentator24** [comment]
oh got it. ya just need to bkb + lotus then attack nightmared bane it will bug out.

i think they should just separate the skill to fix it. 4th skill to end active nightmare at anytime or something.

**Comment by AramilRey** [comment]
This just happened to me on a ranked game on v7.35c, lost the use of the ability for the remainder of the game, match id: 7609775630

Don't know by which mechanism it bugged out, but nothing I tried could revert it

**Comment by ClassTerr** [comment]
Yep, still the issue in 2025. Minus skill for the rest of the game after this happens.

**Comment by aturret** [comment]
This happened again :(
replay: 8622748583

24:53

<img width="2189" height="845" alt="Image" src="https://github.com/user-attachments/assets/729c4f49-899f-4a4b-96f6-a864bd0183a3" />
